### ISSUE TITLE
Hi there!! to be clear

### ISSUE THREAD

**Issue by beloved321** [SciFracX/FractionalDiffEq.jl#36]
Hi there!!
I think the right usage should be:

```julia
solve(prob, h, NonLinearAlg())
```

For the system of fractional ordinary differential equations, there are many solvers available, for example, Grunwald Letnikov difference method ```GL```, fractional linear multiple-step method ```FLMMBDF```, ```FLMMTrap``` and ```FLMMNewtonGregory```. Also, there are solvers for Caputo-Fabrizio fractional operators such as ```NewtonPolynomial```.

_Originally posted by @ErikQQY in https://github.com/SciFracX/FractionalDiffEq.jl/issues/35#issuecomment-1148029299_

**Comment by beloved321** [comment]
julia> using FractionalDiffEq, Plots (process:9076): GLib-GIO-WARNING **:
14:30:09 <https://www.youtube.com/watch?v=ti16lYst-Bc&t=43200s>.828:
Unexpectedly, UWP app `6760NGPDFLab.PDFX_1.3.15.0_x64__sbe4t8mqwq93a'
(AUMId `6760NGPDFLab.PDFX_sbe4t8mqwq93a!App') supports 32 extensions but
has no verbs julia> function chua!(du, x, p, t) a = 10.725; b = 10.593; c =
0.268; m0 = -1.1726; m1 = -0.7872; du[1] =
a*(x[2]-x[1]-(m1*x[1]+0.5*(m0-m1)*(abs(x[1]+1)-abs(x[1]-1)))) du[2] =
u[1]-u[2]+u[3] du[3] = -b*u[2]-c*u[3] end chua! (generic function with 1
method) julia> α = [0.93, 0.99, 0.92]; julia> x0 = [0.2; -0.1; 0.1]; julia>
h = 0.001 0.001 julia> prob = FODESystem(chua!, α, x0) FODESystem(chua!,
[0.93, 0.99, 0.92], [0.2, -0.1, 0.1]) julia> tn = 200; julia> result =
solve(prob, h, tn, NonLinearAlg()) ERROR: MethodError: no method matching
chua!(::Float64, ::Vector{Float64}, ::Int64) Closest candidates are:
chua!(::Any,
::Any, ::Any, ::Any) at REPL[2]:1 Stacktrace: [1] macro expansion @
C:\Users\BELOVED\.julia\packages\FractionalDiffEq\uk4Ug\src\NonLinear\NonLinear.jl:38
[inlined] [2] macro expansion @ .\simdloop.jl:77 [inlined] [3] macro
expansion @
C:\Users\BELOVED\.julia\packages\FractionalDiffEq\uk4Ug\src\NonLinear\NonLinear.jl:37
[inlined] [4] macro expansion @ .\simdloop.jl:77 [inlined] [5]
solve(prob::FODESystem, h::Float64, tn::Int64, ::NonLinearAlg, L0::Float64) @
FractionalDiffEq
C:\Users\BELOVED\.julia\packages\FractionalDiffEq\uk4Ug\src\NonLinear\NonLinear.jl:34
[6] solve(prob::FODESystem, h::Float64, tn::Int64, ::NonLinearAlg) @
FractionalDiffEq
C:\Users\BELOVED\.julia\packages\FractionalDiffEq\uk4Ug\src\NonLinear\NonLinear.jl:15
[7] top-level scope @ REPL[8]:1 I want to understand where I missed it

**Comment by ErikQQY** [comment]
May I ask for the version of FractionalDiffEq.jl?

You can get this by:

```julia
julia>]
pkg>st FractionalDiffEq
```

**Comment by ErikQQY** [comment]
Very sorry for the inconvenience, I think the latest version 0.2.6 should work, there are a lot of changes compared to the old versions.

**Comment by beloved321** [comment]
Julia or FractionalDiffEq

**Comment by ErikQQY** [comment]
I think you are using FractionalDiffEq.jl 0.2.5, that is a very old version( I have not released for a long time, my apologies🥲), I think updating to 0.2.6 would solve all of the problems.😃

**Comment by beloved321** [comment]
How do i do so 
julia> using Pkg
julia> Pkg.update("FractionalDiffEq")???

**Comment by ErikQQY** [comment]
Yes, update FractionalDiffEq.jl to version 0.2.6, there should be no problems.😄

**Comment by beloved321** [comment]
Yea, Thanks a lot. so this is what I need, I download the v1.8.0 of julia and it worked perfectly. Now I will try to build a fractional order lorenz system. I want to do so in the caputo sense, I believe there is provision for it.
I am so grateful for your responses all the while.

**Comment by ErikQQY** [comment]
It is great to see that FractionalDiffEq.jl is helping! So I will close this issue for now. If you have any other questions about this package, don't hesitate to ping me.😄
